src/chapters/orm-relationships/exercises.rst

```diff
@@ -0,0 +1,27 @@
+Exercises: The Early Bird Gets the ORM!
+=======================================
+
+Now that we have gotten very familiar with our ``coding-events`` application, let's design some additional features.
+As you work on your ``coding-events`` application, you may have been inspired by `Meetup <https://www.meetup.com/>`_!
+One of the cool features that Meetup has is that people can sign up for accounts.
+They can use their Meetup accounts to follow the events they are most interested in and keep track of their calendar of events.
+If you want to add similar features to ``coding-events``, you would need to add a ``Person`` class.
```

Comment:
Passive ---> active edit: "To add similar features to ``coding-events``, you need to add a ``Person`` class."

Reply:
Done!